x-pack/plugins/infra/public/pages/logs/log_entry_rate/sections/anomalies/visualisation.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useMemo } from 'react';
+import { AutoRefresh, StringTimeRange } from '../../use_log_entry_rate_results_url_state';
+import { useKibanaContextForPlugin } from '../../../../../hooks/use_kibana';
+import {
+  ANOMALY_SWIMLANE_EMBEDDABLE_TYPE,
+  AnomalySwimlaneEmbeddableInput,
+} from '../../../../../../../ml/public';
+import { EmbeddableRenderer } from '../../../../../../../../../src/plugins/embeddable/public';
+import { partitionField } from '../../../../../../common/infra_ml';
+
+interface Props {
+  stringTimeRange: StringTimeRange;
+  jobIds: string[];
+  selectedDatasets: string[];
+  autoRefresh: AutoRefresh;
```

Comment:
It could cause a slight discrepancy sometimes, yes.

I did originally try to make this work through time range alone, let me reassess what happened with forcing auto refresh to off.